File: src/test/java/zielu/gittoolbox/config/DecorationPartTypeTest.java
package zielu.gittoolbox.config;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.jupiter.api.Test;

class DecorationPartTypeTest {

  @Test
  void shouldNotContainUnknown() {
    assertThat(DecorationPartType.getValues()).doesNotContain(DecorationPartType.UNKNOWN);
  }
}

File: src/test/kotlin/zielu/gittoolbox/config/DecorationPartTypeTest.kt
package zielu.gittoolbox.config

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

internal class DecorationPartTypeTest {

  @Test
  fun `should not contain UKNOWN type`() {
    assertThat(DecorationPartType.getValues()).doesNotContain(DecorationPartType.UNKNOWN)
  }
}
